packages/letspeppol-sdk-php/src/Resources/ProxyClient.php

```
@@ -0,0 +1,328 @@
+     * ```php
+     * $document = $client->proxy()->getDocument('doc123');
+     * echo "Document type: " . $document['documentType'];
+     * echo "Status: " . $document['status'];
+     * file_put_contents('invoice.xml', $document['ubl']);
+     * ```
+     *
+     * @param string $id Document ID
+     * @return array Document details with UBL content
+     * @throws ApiException When document not found (404)
+     */
+    public function getDocument(string $id): array
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)